staging/src/k8s.io/apiserver/pkg/server/httplog/httplog.go

```diff
@@ -224,8 +224,8 @@ func (rl *respLogger) Hijack() (net.Conn, *bufio.ReadWriter, error) {
 }
 
 // CloseNotify implements http.CloseNotifier
-func (rl *respLogger) CloseNotify() <-chan bool {
-	return rl.w.(http.CloseNotifier).CloseNotify()
+func (rl *respLogger) CloseNotify() <-chan struct{} {
```

Comment:
Can we just remove the CloseNotify method?

`respLogger` is private, so probably it's safe to remove.

Reply:
Removed